protocol/packages/currencies/src/payment/only/mod.rs

```diff
@@ -1,24 +1,32 @@
-use currency::{AnyVisitor, Group, Matcher, MaybeAnyVisitResult, MemberOf};
+use serde::{Deserialize, Serialize};
 
-#[cfg(not(feature = "testing"))]
-pub(crate) use r#impl as impl_mod;
-#[cfg(feature = "testing")]
-pub(crate) use testing as impl_mod;
+use currency::{AnyVisitor, Matcher, MaybeAnyVisitResult, MemberOf};
+use sdk::schemars::JsonSchema;
 
-use crate::PaymentGroup;
+use crate::payment;
 
 #[cfg(not(feature = "testing"))]
-pub(crate) mod r#impl;
+pub(crate) mod impl_mod {
+    include!(concat!(env!("OUT_DIR"), "/payment_only.rs"));
+}
+
 #[cfg(feature = "testing")]
-pub(crate) mod testing;
+#[path = "testing.rs"]
+pub(crate) mod impl_mod;
 
-#[derive(Clone, Copy, Debug, Ord, PartialEq, PartialOrd, Eq)]
-pub struct PaymentOnlyGroup {}
+#[derive(
+    Clone, Copy, Debug, Ord, PartialEq, PartialOrd, Eq, Serialize, Deserialize, JsonSchema,
+)]
+#[serde(deny_unknown_fields, rename_all = "snake_case")]
+#[schemars(crate = "sdk::schemars")]
```

Comment:
[everywhere] what does it bring to the definition? We have been living without it.

Reply:
It removes the need for the `sdk::schemars::{self, JsonSchema}` which makes it cryptic why the module itself is imported via `self`. This makes it more explicit and readable.

Reply:
this is quite opinionated 
the `use` solution is more straightforward - the compiler complains ... and one satisfies it. Nobody takes it as "cryptic" because when removes it - he receives the feedback quickly.

Reply:
Well, I did take it as cryptic, but I agree it is opinionated. I would rather have it explicit as all things, but I'm willing to compromise on this.